aptos-move/block-executor/src/executor.rs

```diff
@@ -807,9 +816,6 @@ where
 
                 if res.is_err() {
                     alert!("Failed to serialize resource group");
-                    // Alert first, then log an error with actual btree, to make sure
-                    // printing it can't possibly fail during alert.
-                    error!("Failed to serialize resource group BTreeMap {:?}", btree);
```

Comment:
`error!` prints in the logs. Does `alert!` also do the same thing?

Reply:
it also increments critical errors counter